client/app/components/admin/CeleryStatus.jsx

```diff
@@ -46,7 +46,14 @@ function parseTasks(tasks) {
 function QueuesTable({ loading, queues }) {
   const columns = ['Name', 'Active', 'Reserved', 'Waiting'].map(c => ({ title: c, dataIndex: c.toLowerCase() }));
 
-  return <Table columns={columns} rowKey="name" dataSource={queues} loading={loading} />;
+  return (
+    <Table
+      columns={columns}
+      rowKey="name"
+      dataSource={queues}
+      loading={{ spinning: loading, indicator: <i className="fa fa-spinner fa-pulse" /> }}
```

Comment:
This is definitely better than Ant's default (ugly) indicator :-)

Can we use [`Spin.setDefaultIndicator`](https://ant.design/components/spin/#Static-Method) to globally set this?